File: 1/code.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        hash = {}
        ret = []
        for i in range(0, len(nums)):
            hash[nums[i]] = i
        for i in range(0, len(nums)):
            tmp = target - nums[i];
            if tmp in hash.keys() and hash[tmp] != i:
                ret.append(i)
                ret.append(hash[tmp])
                break;
        return ret

File: 1/code.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    var i = 0, tmp = 0,
        hash = {},
        ret = [];
    for (i = 0; i < nums.length; i += 1) {
        hash[nums[i]] = i;
    }
    for (i = 0; i < nums.length; i += 1) {
        tmp = hash[target - nums[i]];
        if (tmp && tmp != i) {
            ret.push(i);
            ret.push(tmp);
            break;
        }
    }
    return ret;
};